### DISCUSSION TITLE
Capture GitHub Actions data using GitHub Data source Plugin

### DISCUSSION THREAD

**Discussion by YueSheng6** [grafana/grafana#38118] [Data Sources]
**What would you like to be added**:
The GitHub data source cannot read the pipeline data in the Actions. For the CD pipeline, releasing to different stages, the data is not in the release metrics in the GitHub plugin.

**Why is this needed**:
When we measure Deployment frequency, time to deploy test past rate etc., we need data inside the pipelines in Actions.

**Comment by vidyap254** [comment]
Hi, 

Is this feature added yet? It would be of good help if we could send GitHub Actions data into Grafana via the GitHub Data Source Plugin

**Comment by YueSheng6** [comment]
> Hi,
> 
> Is this feature added yet? It would be of good help if we could send GitHub Actions data into Grafana via the GitHub Data Source Plugin

I haven't tried it after. But we used GitHub API to collect the data and load it into Grafana .

**Comment by rexwangcc** [comment]
Came across from another issue on this topic: https://github.com/grafana/github-datasource/issues/105.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
